dts/arm64/renesas/gen3_common/r8a7795_r8a7796_common.dtsi

```diff
@@ -73,6 +73,27 @@
 			#reset-cells = <1>;
 		};
 
+		gpio5: gpio@e6055000 {
+			compatible = "renesas,rcar-gpio";
+			reg = <0 0xe6055000 0 0x50>;
+			#gpio-cells = <2>;
+			gpio-controller;
+			interrupt-parent = <&gic>;
+			interrupts = <GIC_SPI 9 IRQ_TYPE_LEVEL IRQ_DEFAULT_PRIORITY>;
+			clocks = <&cpg CPG_MOD 907>;
+			status = "disabled";
```

Comment:
Can't see where it is enabled

Reply:
it is enabled inside boards/arm64/renesas_rcar/gen3/common_salvator_xs_h3ulcb.dts file